src/containers/RecentTrades/index.tsx

```diff
@@ -70,13 +72,16 @@ class RecentTradesComponent extends React.Component<RecentTradesProps, State> {
     }
 
     private renderContent = () => {
+        const { isMobileDevice } = this.props;
+
         return this.props.userLoggedIn ?
         (
             <TabPanel
                 panels={this.renderTabs()}
                 onTabChange={this.handleMakeRequest}
                 optionalHead={this.props.intl.formatMessage({ id: 'page.body.trade.header.recentTrades' })}
                 currentTabIndex={this.state.index}
+                dropdown={isMobileDevice}
```

Comment:
Naming is wierd. "isMobileDevice" isn't "dropdown"😅
just pass "isMobileDevice" as it is.